src/deephaven_mcp/client/_session_factory.py

```diff
@@ -292,24 +292,34 @@ async def connect_to_server():
             raise InternalError(
```

Comment:
I don't understand the changes in this file.  On this line, an exception is thrown if enterprise is not available.  Nothing below here will ever trigger an error.